### DISCUSSION TITLE
Extend LV nsm - Grafana still indicates old LV Size

### DISCUSSION THREAD

**Discussion by bayden10** [Security-Onion-Solutions/securityonion#2246] [Unsupported Versions] [answered]
Hi,

Noticed after extending the LV nsm, Grafana still indicates old LV size.

![nsm](https://user-images.githubusercontent.com/22577485/101392623-febbca80-3893-11eb-8ca0-0ac52fb2967a.png)

```
vdisplay
  --- Logical volume ---
  LV Path                /dev/system/nsm
  LV Name                nsm
  VG Name                system
  LV UUID                tDR3LJ-uddr-J3fe-pNgW-f0tx-01Fn-6bwpqS
  LV Write Access        read/write
  LV Creation host, time localhost, 2020-12-04 18:41:19 +0000
  LV Status              available
  # open                 1
  LV Size                225.10 GiB
  Current LE             57626
  Segments               2
  Allocation             inherit
  Read ahead sectors     auto
  - currently set to     8192
  Block device           253:2
```

thx
Jim

**Comment by bayden10** [comment]
Hi,

Checked here /opt/so/conf/grafana/grafana_dashboards/standalone and modified the json.

Modified max and absolute values to reflect the additional 100GB. (i.e. old value "max": "131114444000" new value "max": "231114444000")

  "title": "soict - Disk Used(/)",
  "type": "gauge"
},
{
  "cacheTimeout": null,
  "datasource": "InfluxDB",
  "fieldConfig": {
    "defaults": {
      "custom": {},
      "mappings": [
        {
          "id": 0,
          "op": "=",
          "text": "N/A",
          "type": 1,
          "value": "null"
        }
      ],
      "max": "231114444000",
      "min": 0,
      "nullValueMode": "connected",
      "thresholds": {
        "mode": "absolute",
        "steps": [
          {
            "color": "rgba(50, 172, 45, 0.97)",
            "value": null
          },
          {
            "color": "rgba(237, 129, 40, 0.89)",
            "value": "204891555200.0"
          },
          {
            "color": "rgba(245, 54, 54, 0.9)",
            "value": "218002999600.0"
          }
``

Jim

**Reply by bayden10** [reply]
This did not work after so-grafana-restart, values were replaced by old values, possibly docker_container values being held from install.
Will keep digging but if anyone has any insight please feel free to update this thread.

```
----------
          ID: so-grafana
    Function: docker_container.running
      Result: True
     Comment: Created container 'so-grafana'
     Started: 00:04:09.651098
    Duration: 2460.06 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      db388040f8393878dd30c000000000097e4173d60670ee607e9bd90b33d276e2
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-grafana_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 00:04:12.111689
    Duration: 17.003 ms
     Changes:

Summary for local
-------------
Succeeded: 13 (changed=1)
Failed:     0

```

Thx
Jim

**Comment by GeorgiaM-honestly** [comment, marked as answer]
I just had this same issue for shrinking my /nsm. After reboot, the max size in graphana went back to what it had been at install time.

**Reply by bayden10** [reply]
Resolution found here #2222
